### PR TITLE
DocFormCommand: serial doc load&save

### PR DESCRIPTION
**Pull request by msladek**
https://synjira.atlassian.net/browse/CELDEV-899

### CODE REVIEW

File: src/main/java/com/celements/docform/DocFormCommand.java [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+ * that they are correctly prepared for save. IMPORTANT: use exactly ONE instance of this
+ * class for each XWikiContext (each request).
+ *
+ * @author Fabian Pichler <fabian.pichler@synventis.com>
+ * @author Marc Sladek <marc@sladek.me>
+ */
+@Component
+@NotThreadSafe
+@InstantiationStrategy(PER_LOOKUP)
+public class DocFormCommand implements IDocForm {
```
Comment (fpichler) [inline src/main/java/com/celements/docform/DocFormCommand.java:77]: this is no Command anymore. Thus please rename to DocFormService or DocFormDefault or similar.

**Review by fpichler — CHANGES_REQUESTED** [review]

File: src/main/java/com/celements/docform/DocFormCommand.java (hunk 1 above)
Comment (msladek) [reply src/main/java/com/celements/docform/DocFormCommand.java:77]: The class still conforms to the command pattern, this behaviour hasn't changed with this PR.

The object instantiation is now handled by the `ComponentManager` with `@InstantiationStrategy(PER_LOOKUP)` for dependency management, but that doesn't qualify the class as a Service imho.

Comment (fpichler) [reply src/main/java/com/celements/docform/DocFormCommand.java:77]: ok. I see your point: https://alvinalexander.com/java/java-command-design-pattern-in-java-examples/

**Review by fpichler — DISMISSED** [review]

**Comment by msladek** [comment]
resolved merge conflicts

**Review by fpichler — APPROVED** [review]

**Pull request merged**